packages/tees-reporter/utils/helper.js

```diff
@@ -50,8 +50,28 @@ const promiseReadfile = (path) => {
   })
 }
 
+const formatDate = (timesteamp) => {
```

Comment:
It may be more concise.
```javascript
const formatDate = (timestamp) => 
	new Date(timestamp - new Date().getTimezoneOffset() * 60 * 1000)
		.toISOString()
		.slice(0, 19)
		.replace('T', ' ');
```

Reply:
cool 🕶️